docs/examples/video/video_label_example.py

```diff
@@ -51,22 +49,28 @@ def define_graph(self):
         return output
 
 if __name__ == "__main__":
-    pipe = VideoPipe(batch_size=BATCH_SIZE, num_threads=2, device_id=0, data=VIDEO_FILES)
+    pipe = VideoPipe(batch_size=BATCH_SIZE, num_threads=2, device_id=0, data=VIDEO_FILE_ROOT)
     pipe.build()
     for i in range(ITER):
         print("Iteration " + str(i))
-        pipe_out = pipe.run()
-        sequences_out = pipe_out[0].as_cpu().as_array()
-        print(sequences_out.shape)
+        sequences_out, label = pipe.run()
+        sequences_out = sequences_out.as_cpu().as_array()
+        label  = label.as_cpu().as_array()
+        print("sequences shape: ", sequences_out.shape)
+        print("labels shape: ", label.shape)
         print("Got sequence " + str(i*COUNT) + " " + str((i + 1)*COUNT - 1))
+
         for b in range(BATCH_SIZE):
             batch_sequences = sequences_out[b]
+            print(label[b])
             print(batch_sequences.shape)
+            save_dir = 'extracted_frames/' + str(label[b][0]) + '/'
+            os.makedirs(save_dir, exist_ok=True)
```

Comment:
Python2.7 doesn't have `exist_ok` arg supported

Reply:
Updated the PR